README.md

```diff
@@ -17,49 +17,79 @@ Launch files for Ignition Gazebo working with ROS2 control.
 ### `rosbot_controller`
 ROS2 hardware controllers configuration for ROSbots.
 
-## Demo
-Bellow you can find demos with ROSbots:
-- in [rosbot-docker](https://github.com/husarion/rosbot-docker/tree/ros2) you will find a simple example how to use ROSbot.
-- in [rosbot-mapping](https://github.com/husarion/rosbot-mapping) you will find an example how to use ROSbot with the [slam_toolbox](https://github.com/SteveMacenski/slam_toolbox/).
-- in [rosbot-navigation](https://github.com/husarion/rosbot-navigation) you will find an example how to use ROSbot with the [navigation2](https://github.com/ros-planning/navigation2) stack.
+## Source build
 
-# Build
-### Prerequirements
-```bash
-mkdir -p rosbot_ws/src
-cd rosbot_ws
-git clone https://github.com/husarion/rosbot_ros src/
+### Prerequisites
+
+Install `colcon`, `vsc` and `rosdep`:
+```
 sudo apt-get update
 sudo apt-get install -y python3-colcon-common-extensions python3-vcstool python3-rosdep
 ```
 
-### Build
-Inside `rosbot_ws` catalog:
-```bash
+Create workspace folder and clone `rosbot_ros` repository:
+```
+mkdir -p ros2_ws/src
+cd ros2_ws
+git clone https://github.com/husarion/rosbot_ros src/
+```
+
+### Build and run on hardware
+
+Building:
+```
+export HUSARION_ROS_BUILD=hardware
+
+source /opt/ros/$ROS_DISTRO/setup.bash
+
+vcs import src < src/rosbot/rosbot_hardware.repos
+
+rm -r src/rosbot_gazebo
+
+rosdep init
+rosdep update --rosdistro $ROS_DISTRO
+rosdep install -i --from-path src --rosdistro $ROS_DISTRO -y
+colcon build
+```
+Sourcing:
+```
+source install/setup.bash
+```
+Running:
+```
+ros2 launch rosbot_bringup bringup.launch.py
+```
+
+### Build and run Gazebo simulation
+
+Building:
+```
+export HUSARION_ROS_BUILD=simulation
+
+source /opt/ros/$ROS_DISTRO/setup.bash
+
 vcs import src < src/rosbot/rosbot_hardware.repos
 vcs import src < src/rosbot/rosbot_simulation.repos
+
 rosdep init
-rosdep update --rosdistro humble
-rosdep install -r --from-paths src --ignore-src --rosdistro humble -y
-source /opt/ros/humble/setup.bash
+rosdep update --rosdistro $ROS_DISTRO
+rosdep install -i --from-path src --rosdistro $ROS_DISTRO -y
 colcon build
 ```
 
-# Source and run
-### Source
-Inside `rosbot_ws` catalog:
-```bash
+Sourcing:
```

Comment:
I think that it will be nicer to combine it with running, so that it will be only:

Running:
```
source install/setup.bash
ros2 launch rosbot_gazebo simulation.launch.py
```